### DISCUSSION TITLE
was wondering if anyone has a list of [emotions] prompt for Tortoise TTS

### DISCUSSION THREAD

**Discussion by SerAlanTheBold** [neonbjb/tortoise-tts#438] [General]
what emotions is working for now. do Tortoise has example [Sighs] or maybe more? so far the devs only include [I am Really Sad] as an example but did not list all the available emotions.

**Comment by G-force78** [comment]
Yes would realy like to know this too and how far prompting can go, such as is it possible to emphasise words similar to how it can be done in stable diffusion? Also how to have the speaker pause and how to have the speaker say sentence more naturally

**Comment by rancortooth** [comment]
I would really like to be able to introduce specific emotions into the generated speech. I noticed that if I precede a sentence with something like "[I am very angry,]..." then it does generate phrases which sound just ever so slightly more perturbed. No where near "angry" though.

A related point -- I'd also like to be able to control where emphasis is placed in a sentence. 
e.g.: given a sentence such as "I can't believe she's coming with us!", I might want it to be spoken as "I can't believe SHE'S coming with us!" However, tortoise seems to place emphasis at almost random places. Sometimes the result is favorable and other times it makes no sense.

**Comment by Pandaily591** [comment]
Your best bet would be to generate separate voices with clips displaying the emotions you want. Probably.

Also, this is just an idea, but it may be possible to pass the audio through another model that takes in the text and applies emphasis to different parts of the sentence. It just so happens that words/characters seem to correspond to a set amount of mel_tokens, which means you know roughly where in a clip the word was spoken.

**Comment by CoderCowMoo** [comment]
I would suppose that there is no definitive list of emotions that work, and some might work in some contexts but not in others. I guess try to imagine if whatever text you want read, could exist on the internet in whatever tone you're trying to capture. Otherwise you can finetune with hand picked expressiveness from audiobooks perhaps.
